konfera/event/views.py

```diff
@@ -353,3 +350,69 @@ def render_to_response(self, context):
             return redirect('order_detail', order_uuid=self.object.uuid)
 
         return response
+
+
+class CheckInAccessMixin:
+    def dispatch(self, request, *args, **kwargs):
+        if not request.user.groups.filter(name='Checkin').exists():
+            raise Http404
+
+        return super().dispatch(request, *args, **kwargs)
+
+
+class CheckInView(CheckInAccessMixin, ListView):
+    template_name = 'konfera/checkin/list.html'
+    paginate_by = 50
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        context['event'] = Event.objects.get(slug=self.kwargs.get('slug'))
+        context['requested_only'] = 'requested_only' in self.request.GET
+        return context
+
+    def get_queryset(self):
+        event = Event.objects.get(slug=self.kwargs.get('slug'))
+        queryset = Ticket.objects.filter(type__event=event).order_by(Lower('last_name'), Lower('first_name'))
+
+        search = self.request.GET.get('search')
+        if search:
+            queryset = queryset.filter(
+                Q(first_name__icontains=search) |
+                Q(last_name__icontains=search) |
+                Q(email__icontains=search)
+            )
+
+        requested_only = self.request.GET.get('requested_only')
+        if requested_only:
+            queryset = queryset.filter(status=Ticket.REQUESTED)
```

Comment:
How about we add Registered checkbox? Or change this one.

Original idea behind ticket statuses is this:
* requested - user just fills the order, or requested donation for the ticket but hasnt paid yet or donation hasnt been approved yet (i can imagine due to human error this people show up at registration).
* registered - user has paid for the ticket or the donation has been approved
* checked-in - user is on the premise of the conference
* cancelled - user either has not paid after some time, or paid and has canecelled and we have returned the payment.